src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/HttpGoogleServicesDAOSpec.scala

```diff
@@ -329,6 +329,30 @@ class HttpGoogleServicesDAOSpec extends FlatSpec with Matchers with IntegrationT
     }
   }
 
+  it should "calculate data size" in {
+    val random = scala.util.Random
+    val testUser = testCreator.copy(userSubjectId = random.nextLong().toString)
+    val googleWorkspaceInfo = Await.result(gcsDAO.setupWorkspace(testUser, testProject, UUID.randomUUID.toString, testWorkspace, None), Duration.Inf)
```

Comment:
As I said with face-words, you don't need a workspace to test this, just a bucket with some stuff in it.